bundle/manifests/external-dns-operator_clusterserviceversion.yaml

```diff
@@ -119,3 +122,23 @@ spec:
   provider:
     name: Red Hat, Inc.
   version: 0.0.1
+  webhookdefinitions:
+    - admissionReviewVersions:
+        - v1
+      deploymentName: external-dns-operator
+      failurePolicy: Fail
+      generateName: vexternaldns.kb.io
+      containerPort: 9443
+      rules:
+        - apiGroups:
+            - externaldns.olm.openshift.io
+          apiVersions:
+            - v1alpha1
+          operations:
+            - CREATE
+            - UPDATE
+          resources:
+            - externaldnses
+      sideEffects: None
+      type: ValidatingAdmissionWebhook
+      webhookPath: /validate-externaldns-olm-openshift-io-v1alpha1-externaldns
```

Comment:
This path is generated from the comment marker but still, wouldn't it be more approachable to make it shorter?
If you would like to test the webhook server manually, for instance (rare but still possible case).

Reply:
Unfortunately, which registering the webhook in the operator the path is generated:

https://github.com/kubernetes-sigs/controller-runtime/blob/e1880f5026f6e2d8968b64c05cb66b91bce383ec/pkg/builder/webhook.go#L141-L149

And this cannot be overriden easily unless we don't use the helper functions.